airgun/views/common.py

```diff
@@ -385,12 +384,10 @@ class TemplateEditor(View):
         editor = View.nested(TemplateEditor)
 
     """
-    ROOT = ".//div[@class='editor-container']"
-    rendering_options = ToggleRadioGroup(".//div[@class='btn-group']")
-    import_template = Text(".//a[normalize-space(.)='Import']")
-    fullscreen = Text(".//a[normalize-space(.)='Fullscreen']")
-    syntax_type = Select(id='mode')
-    key_binding = Select(id='keybinding')
+    ROOT = ".//div[@id='editor-container']"
+    rendering_options = ItemsList(".//div[contains(@class,'navbar-editor')]/ul")
+    import_template = Text(".//button[@id='import-btn']")
+    fullscreen = Text(".//button[@id='fullscreen-btn']")
```

Comment:
Wonder why we model these buttons using `Text` instead of `Button`. with `Button`, you would be able to do `Button(id='import-btn')`.

Reply:
Nice catch. I just rewrote the id and didn't noticed that there was `Text` not `Button`, but I tested it and it works :smile: . I will update this.